### PR TITLE
Update manual setup instructions to be in line with automated script

### PR DESCRIPTION
**Pull request by LukaszDygon**
Few places where the scripts and manual installation instructions diverged.
I've managed to set the services up I believe, but there may be some things missing/misunderstood about the required setup steps.

### CODE REVIEW

File: MANUAL_INSTALL.md [hunk 1]
```diff
@@ -40,39 +40,54 @@ these steps:
...
+            docker-compose exec -T epb-register-api bash -c 'cd /app && RACK_ENV=production DISABLE_DATABASE_ENVIRONMENT_CHECK=1 make setup-db'
+            ```
 
             Note: The next process can take up to 20 minutes, even on modern
             hardware.
 
-            `docker-compose exec epb-register-api bash -c 'cd /app && bundle
-            exec rake maintenance:import_postcode'`
+            ```sh
+            docker-compose exec epb-register-api bash -c 'cd /app && bundle exec rake maintenance:import_postcode'
```
Comment (LukaszDygon) [inline MANUAL_INSTALL.md:73]: This outputs `Please set the bucket_name or instance_name environment variable`. I recon this is either a missing piece of setup or not required here. Please advise.

File: MANUAL_INSTALL.md [hunk 2]
```diff
@@ -40,39 +40,54 @@ these steps:
...
 
-            `docker-compose exec epb-register-api bash -c 'cd /app && bundle
-            exec rake dev_data:generate_schemes'`
+            docker-compose exec epb-register-api bash -c 'cd /app && bundle exec rake dev_data:generate_assessors'
+            ```
+        3. Setup Frontend
 
-            `docker-compose exec epb-register-api bash -c 'cd /app && bundle
-            exec rake dev_data:generate_assessors'`
+            `docker-compose exec -T epb-frontend bash -c 'cd /app && npm install && make frontend-build'`
```
Comment (cschiragb) [inline MANUAL_INSTALL.md:83]: Not sure if the back-ticks, ` ,are required here?

File: MANUAL_INSTALL.md (hunk 1 above)
Comment (barryhalper) [reply MANUAL_INSTALL.md:73]: I think this is not required as the bucket_name or instance_name refers to s3 buckets used by this rake only when run in PaaS. Locally this does not mean anything. I think therefore you can ignore this

File: MANUAL_INSTALL.md (hunk 2 above)
Comment (LukaszDygon) [reply MANUAL_INSTALL.md:83]: This is just for formatting since it's single line of code

**Pull request merged**